app/views/hyrax/file_sets/show.html.erb

```diff
@@ -11,8 +11,10 @@
         <%= render 'file_set_title', presenter: @presenter %>
       </header>
 
-      <%# TODO: render 'show_descriptions' See https://github.com/samvera/hyrax/issues/1481 %>
-      <%= render 'show_details' %>
+      <% @presenter.show_partials.each do |partial_path| %>
+      <%= render partial_path %>
+      <% end %>
```

Comment:
```suggestion
      <% @presenter.show_partials.each do |partial_path| %>
        <%= render partial_path %>
      <% end %>
```

Reply:
@alishaevn i don't think we want to nest indentation here? that will result in the partial HTML content being indented a level too far (since the `#each` block won't appear in the rendered HTML).